ddtrace/contrib/internal/openai/_endpoint_hooks.py

```diff
@@ -189,7 +189,7 @@ class _CompletionHook(_BaseCompletionHook):
 
     def _record_request(self, pin, integration, span, args, kwargs):
         super()._record_request(pin, integration, span, args, kwargs)
-        if integration.is_pc_sampled_llmobs(span):
+        if integration.llmobs_enabled:
```

Comment:
Do we need to set span kind here if there is [this](https://github.com/DataDog/dd-trace-py/blob/99f03d8016562b4a3ac98a92494bbfe862e0768d/ddtrace/llmobs/_integrations/openai.py#L150C1-L151C1) line that sets the span kind tag based on the operation name?

Reply:
Great catch, thanks!